digits/model/tasks/torch_train.py

```diff
@@ -136,6 +135,8 @@ def task_arguments(self, resources, env):
                 '--save=%s' % self.job_dir,
                 '--snapshotPrefix=%s' % self.snapshot_prefix,
                 '--snapshotInterval=%s' % self.snapshot_interval,
+                '--mean=%s' % self.dataset.path(constants.MEAN_FILE_IMAGE),
+                '--labels=%s' % self.dataset.path(self.dataset.labels_file),
```

Comment:
Datasets don't necessarily have a `labels_file`. That's why you're getting these TravisCI errors for generic Torch tests:
```
======================================================================
FAIL: digits.model.images.generic.test_views.TestTorchCreation.test_create_wait_delete
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/miniconda/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/travis/build/NVIDIA/DIGITS/digits/model/images/generic/test_views.py", line 245, in test_create_wait_delete
    assert self.model_wait_completion(job_id) == 'Done', 'create failed'
AssertionError: create failed
-------------------- >> begin captured logging << --------------------
digits: ERROR: AttributeError: 'GenericImageDatasetJob' object has no attribute 'labels_file'
--------------------- >> end captured logging << ---------------------
```